control-plane/agents/core/tests/pool/mod.rs

```diff
@@ -770,7 +770,11 @@ async fn disown_unused_replicas() {
         .unwrap();
 
     let volume = volumes_api
-        .put_volume_target(&volume.spec.uuid, &node, models::VolumeShareProtocol::Nvmf)
+        .put_volume_target(
+            &volume.spec.uuid,
+            models::VolumeShareProtocol::Nvmf,
+            Some(&node),
```

Comment:
why is the order different now?

Reply:
Yeah, this is what I even noticed, I am not sure, somehow the openapi generator generated code changed the order after making the node field not required.